packages/i18n/package.json

```diff
@@ -26,7 +26,9 @@
   "exports": {
     "default": "./src/index.js"
   },
-  "main": "./src/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist-module/index.js",
+  "source": "./src/index.js",
```

Comment:
Was [source](https://parcel2-docs.vercel.app/configuration/package-json/#source) added for parcel? I wonder why Gutenberg doesn't have source in their `package.json`

Reply:
Okay, I see it's being used in `rollup.config.js`